src/main/java/edu/cornell/kfs/pmw/batch/service/impl/PaymentWorksVendorToKfsVendorDetailConversionServiceImpl.java

```diff
@@ -660,6 +809,14 @@ private KfsVendorDataWrapper createConflictOfInterestNote(PaymentWorksVendor pmw
         return kfsVendorDataWrapper;
     }
 
+    private KfsVendorDataWrapper createInsuranceCertificateNote(PaymentWorksVendor pmwVendor, KfsVendorDataWrapper kfsVendorDataWrapper) {
+        if (paymentWorksVendorIsPurchaseOrderVendor(pmwVendor) && StringUtils.isNotBlank(pmwVendor.getInsuranceCertificate())) {
+            StringBuilder sbText = new StringBuilder(getConfigurationService().getPropertyValueAsString(PaymentWorksKeyConstants.NEW_VENDOR_PVEN_NOTES_INSURANCE_CERTIFICATE_URL_EXISTS_MESSAGE));
```

Comment:
Nothing is being appended to this StringBuilder after constructing it with the initial text, so this could be replaced with a plain String instead.

Reply:
fixing